app/src/main/res/drawable/ic_quest_hairdresser.xml

```diff
@@ -0,0 +1,35 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" xmlns:aapt="http://schemas.android.com/aapt"
```

Comment:
You should include the SVG source graphic for the quest too

Reply:
added